### PR TITLE
Untitled Pull Request

### PR DESCRIPTION
**Pull request by biennviquiera**
Created login view controller outlets

### CODE REVIEW

File: Disher/LoginViewController.m [hunk 1]
```diff
@@ -16,8 +20,18 @@ @implementation LoginViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     // Do any additional setup after loading the view.
+    
+    
+
```
Comment (melissa-gu) [inline Disher/LoginViewController.m]: nit: remove excess whitespace here/elsewhere

File: Disher/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -44,18 +44,28 @@
...
+                                    <action selector="didTapSignup:" destination="BYZ-38-t0r" eventType="touchUpInside" id="wzB-jF-jHn"/>
+                                </connections>
                             </button>
                         </subviews>
                         <viewLayoutGuide key="safeArea" id="6Tk-OE-BBY"/>
                         <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                     </view>
+                    <connections>
+                        <outlet property="passwordField" destination="FhQ-sL-LVB" id="GQl-IH-n2R"/>
+                        <outlet property="usernameFIeld" destination="N76-I4-e2g" id="4V5-hb-gmb"/>
```
Comment (melissa-gu) [inline Disher/Base.lproj/Main.storyboard]: nit: `usernameField` instead of `usernameFIeld`

**Review by melissa-gu — CHANGES_REQUESTED** [review]

File: Disher/Base.lproj/Main.storyboard (hunk 2 above)
Comment (biennviquiera) [reply Disher/Base.lproj/Main.storyboard]: comment

**Review by melissa-gu — CHANGES_REQUESTED** [review]
these are nits, but requesting changes so we can practice the flow

**Comment by melissa-gu** [comment]
I think there was some additional logic added in the commit but lgtm!

For the future let's add a title and description summarizing the added functionality to PRs. If applicable you can add screenshots/videos (maybe using pixelcloud) which help with reviewing

**Review by melissa-gu — APPROVED** [review]

**Pull request merged**